src/bulls_and_cows_lib/board.cpp

```diff
@@ -4,5 +4,41 @@
 namespace bulls_and_cows {
 
     // TODO: define the body of the functions declared in board.cpp
+    // Create a board and with a new random secret code composed of allowed characters
+    Board create_board(const GameOptions& game_options)
+    {
+        //game_option =
+    }
+
+    // Validate that a user-entered code contains proper number of allowed characters
+    bool validate_attempt(const GameOptions& game_options, const Code& attempt)
+    {
```

Comment:
In the body of this function, you must :
- check if attempt.value has a number of characters equal to game_options.number_of_characters_per_code
- check if attempt.value has all his characters between game_options.minimum_allowed_character and game_options.maximum_allowd_character
- if any of these conditions is false then you return false, else true